crates/common/tedge_config/src/tedge_config_cli/new.rs

```diff
@@ -460,6 +462,10 @@ define_tedge_config! {
             /// The default software plugin to be used for software management on the device
             #[tedge_config(example = "apt")]
             default: String,
+
+            /// The maximum number of software packages reported by sm-plugin
```

Comment:
This message is for the end-user who is unaware of any `sm-plugin`.

```suggestion
            /// The maximum number of software packages reported for each type of software package
```